### PR TITLE
Spark application logger.

### PR DESCRIPTION
**Pull request by rabarona**
Aims to fix issue DB-520 Logger does not work in ML pipeline.
Users now are able to see what is going on with their oni-ml application.

screen shot 2016-09-21 at 3 17 49 pm

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,15 +4,14 @@ version := "1.1"
 
 scalaVersion := "2.10.6"
 
-import AssemblyKeys._
+import sbtassembly.Plugin.AssemblyKeys._
```
Comment (NathanSegerlind) [inline build.sbt]: what's the story with this change?

Comment (rabarona) [reply build.sbt]: Good catch. I don't know how that change got there. I will revert that. Thanks.

Comment (rabarona) [reply build.sbt]: Looks like something my IDE did. Just added the package name to AssemblyKeys._

**Comment by NathanSegerlind** [comment]
looks good to me!

**Pull request merged**